install-dependencies.sh

```diff
@@ -70,11 +70,6 @@ fedora_packages=(
     git
     python
     sudo
-    java-1.8.0-openjdk-headless
-    java-1.8.0-openjdk-devel
-    ant
-    ant-junit
-    maven
     patchelf
```

Comment:
We need Java for antlr3. But likely antlr3 pulls its own java.

Reply:
yeah, it requires java-headless. unless we want to specify the exact java version to fulfill this requirement, we don't need to specify which java-headless version we need here. please note, java-17-openjdk-headless also provides java-headless.